antora/docs/modules/ROOT/nav.adoc

```diff
@@ -1,6 +1,6 @@
 * xref:index.adoc[Overview]
-* xref:ec-cli:ROOT:ec.adoc[EC CLI]
-* xref:ec-cli:ROOT:verify-enterprise-contract.adoc[EC Task]
-* xref:ec-policies:ROOT:index.adoc[Policies]
+* xref:ec-cli:ROOT:ec_validate_image.adoc[CLI Reference]
+* xref:ec-cli:ROOT:verify-enterprise-contract.adoc[Tekton Tasks]
 * xref:ecc:ROOT:index.adoc[Configuration]
-* xref:ec-cookbook:ROOT:index.adoc[Cookbook]
+* xref:ec-policies:ROOT:release_policy.adoc[Policies]
+* xref:ec-cookbook:ROOT:index.adoc[User Guide]
```

Comment:
The names are inconsistent with the names in the respective antora.yml but we'll worry about that later.